src/main/java/org/zone/region/flag/entity/player/move/message/leaving/LeavingFlagType.java

```diff
@@ -34,23 +37,44 @@
 
     @Override
     public @NotNull LeavingFlag load(@NotNull ConfigurationNode node) throws IOException {
-        String message = node.node("Message").getString();
+        String message = node.node("LeavingMessage").getString();
+        String displayType = node.node("LeavingMessageDisplayType").getString();
```

Comment:
The user knows its the LeavingMessage flag, therefore the prefix ``LeavingMessage`` can be removed

Reply:
mm